tests/examples/logging/playground/printstuff/log/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+exec multilog t s10485760 n5 .
```

Comment:
t = TAI timestamp
s10485760 = rotate logs every 10mb
n5 = keep 5 logs in history